docs/reference/authentication.rst

```diff
@@ -23,15 +23,20 @@ The CLI package includes the ``flocker-ca`` program which is used to generate ce
 
 You can run ``flocker-ca --help`` for a full list of available commands. For API user certificates, run the ``flocker-ca create-api-certificate`` command from the same directory containing the certificate authority files generated when you first :ref:`installed the cluster <authentication>`.
 
-Run ``flocker-ca create-api-certificate <username>``, where ``<username>`` is a unique username for an API user.
+Run ``flocker-ca create-api-certificate <username>``:
 
 .. code-block:: console
 
-   $ flocker-ca create-api-certificate alice
-   Created alice.crt and alice.key. You can now give these to your API end user so they can access the control service API.
+   $ flocker-ca create-api-certificate user
+   Created user.crt and user.key. You can now give these to your API end user so they can access the control service API.
 
-The two files generated will correspond to the username you specified in the command, in this example ``alice.crt`` and ``alice.key``.
-You should securely provide a copy of these files to the API end user, as well as a copy of the cluster's public certificate, the ``cluster.crt`` file.
+.. note:: In this command ``<username>`` is a unique username for an API user.
+   Please note though that ``flocker-deploy`` requires these files to be named :file:`user.crt` and :file:`user.key`.
```

Comment:
This line describing the problem is fine, but it doesn't make sense if we ALSO make people generate them with the name "user" in the first place.